feature-libs/checkout/schematics/add-checkout/index.ts

```diff
@@ -6,31 +6,155 @@ import {
+  SPARTACUS_CHECKOUT_B2B_ROOT,
+  SPARTACUS_CHECKOUT_BASE,
+  SPARTACUS_CHECKOUT_BASE_ASSETS,
+  SPARTACUS_CHECKOUT_BASE_ROOT,
+  SPARTACUS_CHECKOUT_SCHEDULED_REPLENISHMENT,
+  SPARTACUS_CHECKOUT_SCHEDULED_REPLENISHMENT_ASSETS,
+  SPARTACUS_CHECKOUT_SCHEDULED_REPLENISHMENT_ROOT,
+  SPARTACUS_ORDER,
   validateSpartacusInstallation,
 } from '@spartacus/schematics';
 import { peerDependencies } from '../../package.json';
 import {
-  CHECKOUT_FEATURE_NAME_CONSTANT,
+  CHECKOUT_B2B_MODULE,
+  CHECKOUT_B2B_ROOT_MODULE,
+  CHECKOUT_B2B_TRANSLATIONS,
+  CHECKOUT_B2B_TRANSLATION_CHUNKS_CONFIG,
+  CHECKOUT_BASE_FEATURE_NAME_CONSTANT,
+  CHECKOUT_BASE_MODULE,
+  CHECKOUT_BASE_MODULE_NAME,
+  CHECKOUT_BASE_ROOT_MODULE,
+  CHECKOUT_BASE_TRANSLATIONS,
+  CHECKOUT_BASE_TRANSLATION_CHUNKS_CONFIG,
   CHECKOUT_FOLDER_NAME,
-  CHECKOUT_MODULE,
-  CHECKOUT_MODULE_NAME,
-  CHECKOUT_ROOT_MODULE,
-  CHECKOUT_TRANSLATIONS,
-  CHECKOUT_TRANSLATION_CHUNKS_CONFIG,
+  CHECKOUT_SCHEDULED_REPLENISHMENT_MODULE,
+  CHECKOUT_SCHEDULED_REPLENISHMENT_ROOT_MODULE,
+  CHECKOUT_SCHEDULED_REPLENISHMENT_TRANSLATIONS,
+  CHECKOUT_SCHEDULED_REPLENISHMENT_TRANSLATION_CHUNKS_CONFIG,
   SCSS_FILE_NAME,
-  SPARTACUS_CHECKOUT_ASSETS,
-  SPARTACUS_CHECKOUT_ROOT,
 } from '../constants';
 
+const checkoutBaseFeatureConfig: FeatureConfig = {
+  folderName: CHECKOUT_FOLDER_NAME,
+  moduleName: CHECKOUT_BASE_MODULE_NAME,
+  featureModule: {
+    name: CHECKOUT_BASE_MODULE,
+    importPath: SPARTACUS_CHECKOUT_BASE,
+  },
+  rootModule: {
+    name: CHECKOUT_BASE_ROOT_MODULE,
+    importPath: SPARTACUS_CHECKOUT_BASE_ROOT,
+  },
+  lazyLoadingChunk: {
+    moduleSpecifier: SPARTACUS_CHECKOUT_BASE_ROOT,
+    namedImports: [CHECKOUT_BASE_FEATURE_NAME_CONSTANT],
+  },
+  i18n: {
+    resources: CHECKOUT_BASE_TRANSLATIONS,
+    chunks: CHECKOUT_BASE_TRANSLATION_CHUNKS_CONFIG,
+    importPath: SPARTACUS_CHECKOUT_BASE_ASSETS,
+  },
+  styles: {
+    scssFileName: SCSS_FILE_NAME,
+    importStyle: SPARTACUS_CHECKOUT,
+  },
+  dependencyManagement: {
+    featureName: CLI_CHECKOUT_BASE_FEATURE,
+    featureDependencies: {
+      [SPARTACUS_CART]: [CLI_CART_BASE_FEATURE],
+      [SPARTACUS_ORDER]: [CLI_ORDER_FEATURE],
+    },
+  },
+  recreate: true,
+};
+
+const checkoutB2bFeatureConfig: FeatureConfig = {
+  folderName: CHECKOUT_FOLDER_NAME,
+  moduleName: CHECKOUT_BASE_MODULE_NAME,
+  featureModule: {
+    name: CHECKOUT_B2B_MODULE,
+    importPath: SPARTACUS_CHECKOUT_B2B,
+  },
+  rootModule: {
+    name: CHECKOUT_B2B_ROOT_MODULE,
+    importPath: SPARTACUS_CHECKOUT_B2B_ROOT,
+  },
+  lazyLoadingChunk: {
+    moduleSpecifier: SPARTACUS_CHECKOUT_BASE_ROOT,
+    namedImports: [CHECKOUT_BASE_FEATURE_NAME_CONSTANT],
+  },
+  i18n: {
+    resources: CHECKOUT_B2B_TRANSLATIONS,
+    chunks: CHECKOUT_B2B_TRANSLATION_CHUNKS_CONFIG,
+    importPath: SPARTACUS_CHECKOUT_B2B_ASSETS,
+  },
+  styles: {
+    scssFileName: SCSS_FILE_NAME,
+    importStyle: SPARTACUS_CHECKOUT,
+  },
+  dependencyManagement: {
```

Comment:
don't think we need this for b2b and repl with the current mechanism under the hood in our root module

Reply:
This part is to make the schematics install cart and order libs.
It's not related to the LL dep mechanism.